Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -548,48 +548,6 @@ for (unsigned i = 0, n = particle_diags.size(); i < n; ++i) {
         pinned_pc->make_alike<amrex::PinnedArenaAllocator>() :
         pc->make_alike<amrex::PinnedArenaAllocator>();
 
-    // names of amrex::Real and int particle attributes in SoA data
```

Comment:
I had to move this code further down, so that it comes after `tmp.AddRealComp("phi");`